### PR TITLE
Room show page redesign

### PR DESCRIPTION
**Pull request by cecyc**
Added cover image to room page.
Reformatted the page to show two columns under the cover page.

### CODE REVIEW

**Comment by jah2488** [comment]
🙌🏻

**Comment by wrpaape** [comment]
:+1:  just waiting on tests

**Comment by mlgiardina** [comment]
fantastique! :+1:

File: app/views/rooms/show.html.erb [hunk 1]
```diff
@@ -1,37 +1,45 @@
...
+      <h3>Meetings</h3>
+      <% if @room.meetings.count == 0 %>
+        <p>No meetings scheduled here.</p>
+      <% else %>
+        <table class="table">
+          <tr>
+            <td><strong>Topic</strong></td>
+            <td><strong>Start time</strong></td>
+            <td><strong>End time</strong></td>
+          </tr>
```
Comment (JMD1986) [inline app/views/rooms/show.html.erb:30]: looks alright to me :+1:

**Pull request merged**